docs/how-it-works.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 2
+---
+
+# How it Works
+
+Learn the high-level workings of Divvi Up and how it accomplishes the
+privacy-preserving property.
+
+## Breakdown
+
+Divvi Up functionality can be broadly broken down into five steps.
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+### Step 1: Client generate measurements
+
+An app developer wants to collect statistics about their app, such as user
+behavior, demographics, or system information. The developer writes app
+functionality that takes measurements of these properties.
+
+In Divvi Up, the developer creates a Task, which encodes data about the
+measurement being taken. This encoded data is stored in the app and used by
+client libraries. The data includes the measurement type, the two Divvi Up
+aggregators being used, the identifier for the task, and how aggregates are to
+be queried. The measurement type includes, but is not limited to, counts, sums,
+and histograms.
+
+For each measurement, the subsequent steps are performed.
+
+### Step 2: Client divides the measurement
+
+Using a client library, the app divides the measurement into two shares, known
```

Comment:
I know this is a hard task and we may have to make several iterative passes at it, but we should make some passing effort at explaining the nature of "dividing" the measurements. We may even want to call it sharding because dividing a scalar has a clear and incorrect meaning here. It's even less intuitive to a naïve reader to consider (by any implicit definition of sharding/splitting/dividing) "splitting a boolean in half"

Reply:
I landed on `s/divides/randomly shards/`.